app/views/spree/admin/orders/bulk_management.html.haml

```diff
@@ -20,12 +20,12 @@
       %label{ :for => 'start_date_filter' }
         = t("admin.start_date")
       %br
-      %input{ :class => "two columns alpha", :type => "text", :id => 'start_date_filter', 'ng-model' => 'startDate', 'datepicker' => "startDate", 'confirm-change' => "confirmRefresh()", 'ng-change' => 'refreshData()' }
+      %input{ :class => "two columns alpha", :type => "text", :id => 'start_date_filter', 'ng-model' => 'startDate', 'datepicker' => "startDate", 'confirm-change' => "confirmRefresh()", 'ng-change' => 'refreshData()', 'ng-model-options' => '{ debounce: 1000 }' }
```

Comment:
the other option would be to make this as the products page were the user needs to click "search" after they select all filters.

Reply:
I'd love to see a bit more UX unification on the filtering in BOM, Orders and Products :pray:

Reply:
UX unification for BOM, Orders and Products would be awesome :+1: I guess it needs some planning and inception though.